verticapy/_config/config.py

```diff
@@ -148,6 +153,205 @@ def set_option(key: str, value: Any = None) -> None:
     value: object, optional
         New value of the option.
+
+    Examples
+    --------
+    Import and load the titanic dataset:
+
+    .. hint:: VerticaPy provides multiple datasets, all of which have loaders in the datasets module.
+
+    .. code-block:: python
+
+        from verticapy.datasets import load_titanic
+
+        titanic = load_titanic()
+        display(titanic)
+
+    .. ipython:: python
+        :suppress:
+
+        from verticapy.datasets import load_titanic
+        from verticapy import set_option
+
+        titanic = load_titanic()
+        html_file = open("figures/_config_config_set_option_1.html", "w")
+        html_file.write(titanic._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/_config_config_set_option_1.html
+
+    Import the set_option function:
+
+    .. code-block:: python
+
+        from verticapy import set_option
+
+    Turn on the "count_on" option, which displays the total number of elements in the dataset:
+
+    .. code-block:: python
+
+        set_option("count_on", True)
+        display(titanic)
+
+    .. ipython:: python
+        :suppress:
+
+        set_option("count_on", True)
+        html_file = open("figures/_config_config_set_option_2.html", "w")
+        html_file.write(titanic._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/_config_config_set_option_2.html
+
+    Turn off the display footer:
+
+    .. code-block:: python
+
+        set_option("footer_on", False)
+        display(titanic)
+
+    .. ipython:: python
+        :suppress:
+
+        set_option("footer_on", False)
+        html_file = open("figures/_config_config_set_option_3.html", "w")
+        html_file.write(titanic._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/_config_config_set_option_3.html
+
+    Sets the maximum number of columns displayed:
+
+    .. note:: By setting this parameter, we retrieve fewer elements from the database, resulting in faster visualization.
+
+    .. code-block:: python
+
+        set_option("max_columns", 3)
+        display(titanic)
+
+    .. ipython:: python
+        :suppress:
+
+        set_option("max_columns", 3)
+        html_file = open("figures/_config_config_set_option_4.html", "w")
+        html_file.write(titanic._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/_config_config_set_option_4.html
+
+    Sets the maximum number of rows displayed:
+
+    .. code-block:: python
+
+        set_option("max_rows", 5)
+        display(titanic)
+
+    .. ipython:: python
+        :suppress:
+
+        set_option("max_rows", 5)
+        html_file = open("figures/_config_config_set_option_5.html", "w")
+        html_file.write(titanic._repr_html_())
+        html_file.close()
+
+    .. raw:: html
+        :file: SPHINX_DIRECTORY/figures/_config_config_set_option_5.html
+
```

Comment:
.. warning:: Exercise caution when using high values for these two options, as it may lead to an excessive amount of data being loaded into memory for display. This can potentially slow down your notebook's performance.